.github/workflows/deploy-lnt.llvm.org.yaml

```diff
@@ -0,0 +1,41 @@
+name: Deploy lnt.llvm.org
+
+on:
+  push:
+    paths:
+      - '.github/workflows/deploy-lnt.llvm.org.yaml'
+      - 'deployment/*'
+  pull_request_target:
```

Comment:
@boomanaiden154 Is that the right trigger event? I think we do want the contents of the PR branch to be used for this check though, since otherwise we'd be deploying with whatever `deployment/*` we have on `main`. However, if we do that, then I presume someone malicious could craft a PR to print our secrets, so perhaps that's just not possible?

If that's not possible, then I somewhat wonder what's the point of running `terraform plan` on PRs in the first place, if that's only going to be running with the `main.tf` (& related files) that are already on `main`.